torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -19,6 +29,25 @@ def get_native_backend_config_dict():
         # optional, output activation dtype
         "output_dtype": torch.quint8
     }
+
+    # input_activation: quint8, output_activation: quint8
```

Comment:
nit: make these comments consistent across different dtype configs? Also I feel some of these comments are actually not needed, like this one doesn't give much additional information IMO

Reply:
sounds good, I can remove these